src/components/Dashboard/TodoList.vue

```diff
@@ -255,27 +247,56 @@
                   <span v-else>PAID</span>
                 </div>
 
-                <!-- change-requested filing -->
+                <!-- draft continuation filing -->
```

Comment:
Would it be  possible to move this block above the "other" block?

I think that would simplify the logic a bit. The "other" block can be drafts that aren't already handled in code above it.

Reply:
Sure, moved it above other draft